packages/ui/src/components/Session/Session/Session.tsx

```diff
@@ -58,30 +59,45 @@ export const Session: React.FC<Props> = ({
   currentUserId,
   chat,
 }) => {
+  const mq = useMediaQuery();
+
   return (
-    <div className="tw-flex tw-flex-col tw-gap-10">
+    <div className="tw-flex tw-flex-col tw-h-full tw-gap-4 lg:tw-gap-10 tw-pb-[66px]">
       <div
         className={cn(
           "tw-w-full tw-aspect-video tw-grow tw-border tw-border-brand-700 tw-bg-brand-100",
-          "tw-rounded-lg tw-overflow-hidden tw-flex tw-flex-row tw-gap-6"
+          "tw-rounded-lg tw-overflow-hidden",
+          chat.enabled
+            ? "lg:tw-grid tw-relative lg:tw-grid-cols-[auto,minmax(35%,326px)]"
+            : "tw-flex"
         )}
       >
-        <SessionStreams
-          currentUserId={currentUserId}
-          fullScreen={fullScreen}
-          streams={streams}
-          chat={chat.enabled}
-          timer={timer}
-          alert={alert}
-        />
-
         <AnimatePresence mode="wait">
-          {chat.enabled ? (
-            <AnimateWidth key="chat">{chatPanel}</AnimateWidth>
-          ) : null}
+          <AnimateWidth className="!tw-w-full">
+            <SessionStreams
+              currentUserId={currentUserId}
+              fullScreen={fullScreen}
+              streams={streams}
+              chat={chat.enabled}
+              timer={timer}
+              alert={alert}
+            />
+          </AnimateWidth>
         </AnimatePresence>
+
+        {chat.enabled ? (
+          mq.lg ? (
+            <AnimatePresence mode="wait">
+              <AnimateWidth key="chat" className="tw-h-full">
+                {chatPanel}
+              </AnimateWidth>
+            </AnimatePresence>
+          ) : (
+            chatPanel
+          )
+        ) : null}
```

Comment:
Nested ternary operator are not prefared. 

```suggestion
        {chat.enabled &&  mq.lg ? (
            <AnimatePresence mode="wait">
              <AnimateWidth key="chat" className="tw-h-full">
                {chatPanel}
              </AnimateWidth>
            </AnimatePresence>
        ) : null}
        
        {
         chat.enabled && !mq.lg ? chatPannel : null 
        }
```